File: c++/fact.cpp
//factorial of a number using for loop
#include <iostream>
using namespace std;
int main()
{
    long long int n, fact = 1;
    cout << "Enter a positive integer: ";
    cin >> n;
    for(int i = 1; i <= n; ++i)
    {
        fact *= i;
    }
    cout << "Factorial of " << n << " = " << fact;    
    return 0;
}

File: java/Fact.java
import java.util.Scanner;
class Fact{
    public static void main(String[] a){
        Scanner s = new Scanner(System.in);
        System.out.println("enter num ");
        int x = s.nextInt();
        int y = 1;

        for(int i=1;i<=x;i++){
            y = y*i;
        }
        System.out.println("factorial is " + y);
}}